setup.cfg

```diff
@@ -15,16 +15,17 @@ include_package_data = True
 packages = find:
 platforms = any
 install_requires =
-    aiohttp==3.9.3
+    aiobotocore==2.12.0
```

Comment:
The latest `aiobotocore` is not compatible with `neuro-cli`. You will receive following error when you upgrade `aiobotocore`:
```
'AIOHTTPSession' object has no attribute '_session'
```